packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -1035,6 +1040,9 @@ class Dropdown extends AutoControlledComponent<Extendable<DropdownProps>, Dropdo
 
     if (getA11ySelectionMessage && getA11ySelectionMessage.onRemove) {
       this.setState({ a11ySelectionStatus: getA11ySelectionMessage.onRemove(poppedItem) })
+      setTimeout(() => {
+        this.setState({ a11ySelectionStatus: '' })
+      }, Dropdown.a11yStatusCleanupTime)
```

Comment:
Just note: we should properly cleanup these timeouts.

Reply:
like with a debounce or what do you mean?

Reply:
```
      clearTimeout(this.a11yStatusTimeout)
      this.a11yStatusTimeout = setTimeout(() => {
        this.setState({ a11ySelectionStatus: '' })
      }, Dropdown.a11yStatusCleanupTime)

//

componentWillUnmount() {
  clearTimeout(this.a11yStatusTimeout)
}
```

Or you will randomly have this warning:
```
Warning: Can’t call setState (or forceUpdate) on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in the componentWillUnmount method.
```

https://medium.freecodecamp.org/how-to-work-with-react-the-right-way-to-avoid-some-common-pitfalls-fc9eb5e34d9e

See `Clear Subscriptions` part 👍

Reply:
oh cool. will integrate this. thank you!